modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,59 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs.
+// Has this changed recently?
+The Data Mover solution uses the Restic option of VolSync.
 
-[NOTE]
-=====
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2.x Data Mover VolumeSnapshotBackups (VSBs) and VolumeSnapshotRestores (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved, by specifying a concurrent number of VSBs and VSRs simultaneously in Progress. After all async plugin operations are completed, the backup is marked as completed.
+
+
+[NOTE]
+====
+The OADP 1.2.x Data Mover, is a fully supported Red Hat feature.
+
+The OADP 1.1.x Data Mover, is a Technology Preview feature.
+====
+:FeatureName: The OADP 1.1.x Data Mover
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+Red Hat recommends customers use OADP 1.2.x Data Mover to back up and restore ODF CephFS volumes, and upgrade or install OCP 4.12 for improved performance. OADP Data Mover can leverage CephFS shallow volumes in OCP 4.12+, which based on our testing improves the performance of backup times. For more information about OADP 1.2.x with CephS [name of topic], see ___.
```

Comment:
s/volumes, and/volumes/g
please include the referenced links to the jira issue and upstream design that I had posted

Reply:
@weshayutin I updated the note and added the links.